components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -490,6 +493,10 @@ public RefreshTokenValidationDataDO validateRefreshToken(String consumerKey, Str
                     "TOKEN_SCOPE, TOKEN_STATE, TIME_CREATED FROM " + accessTokenStoreTable +
                     " WHERE CONSUMER_KEY = ? AND REFRESH_TOKEN = ? ORDER BY TIME_CREATED DESC LIMIT 1";
 
+            db2Query = "SELECT ACCESS_TOKEN, AUTHZ_USER, " +

```

Comment:
in validateRefreshToken method we get userStoreDomain and append it to accessTokenStoreTable with '_' if it is not null. we can't move queries directly to SQLQueries class.